### PR TITLE
fix linting

### PR DESCRIPTION
**Pull request by bcomnes**
Fix linting.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -86,12 +86,14 @@ function AutoIndex (db, idb, reduce) {
   }
 
   secondary.createValueStream = function (opts) {
-    (opts && opts || (opts = {})).keys = false
+    if (!opts) opts = {}
+    opts.keys = false
     return secondary.createReadStream(opts)
```
Comment (bcomnes) [inline index.js]: @ungoldman can you double check my javascripting here?  This was some weird sort circuit shit that was mind bendy.

File: index.js [hunk 2]
```diff
@@ -86,12 +86,14 @@ function AutoIndex (db, idb, reduce) {
...
   }
 
   secondary.createKeyStream = function (opts) {
-    (opts && opts || (opts = {})).values = false
+    if (!opts) opts = {}
+    opts.values = false
     return secondary.createReadStream(opts)
   }
```
Comment (bcomnes) [inline index.js]: Here too

File: index.js (hunk 1 above)
Comment (ungoldman) [inline index.js]: you want to ensure that (1) opts is an object, and (2) opts.keys is always set to false, right?

Comment (bcomnes) [inline index.js]: I think so, I copied most of this code from https://github.com/juliangruber/level-secondary.  Tests still pass  ¯\_(ツ)_/¯

Comment (ungoldman) [inline index.js]: ok :+1:

**Pull request merged**